playground/mocks/data/idp/idx/authenticator-select-enroll-options.json

```diff
@@ -118,7 +118,7 @@
       {
         "label": "Security Key or Biometric Authenticator (FIDO2)",
         "value": {
-          "type": "webauthn",
+          "type": "security_key",
```

Comment:
Did we decide the type would be 'security_key'?

Reply:
Yeah, type will be security_key per https://github.com/okta/okta-core/blob/a12c55b0a748b3366dd94aa082e7d697bb7dcf2a/server/factors.interface/src/main/java/com/okta/services/factors/iface/types/OrgAuthenticatorType.java#L28 and https://github.com/okta/okta-core/blob/a12c55b0a748b3366dd94aa082e7d697bb7dcf2a/server/core/src/main/java/com/saasure/db/dto/core/entity/authenticators/securitykey/SecurityKeyAuthenticator.java#L23-L26

(Security Key is the authenticator type in which we'll have the WebAuthn method)